### PR TITLE
FE-101: Front-End Foundations

### PR DESCRIPTION
**Pull request by nicoRDL**
## Description

- creates first html page for sample site; a basic article page
- adds basic styling and css to article page
- adds images into the article page

## Related Ticket

https://codeandtheory.atlassian.net/browse/FE-648

## Testing Instructions

1. Pull from FE-101/FE-648 branch
2. Review the files and code included

## Risks

N/A

### CODE REVIEW

File: FE-101/fe101-article-page.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Document</title>
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html:8]: Inconsistent indentation. Please use 2 spaces only.

File: FE-101/fe101-article-page.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+      <section class="opening-section">
+        <header>
+          <h1>Why Mechanical Keyboards are Better For Gaming</h1>
+        </header>
+        <div class="opening-para">
+          <p>Mechanical keyboards are great to use for gaming for a multitude of reasons and are incredibly fun to play with. While mechanical keyboards are slightly more expensive than traditional keyboards, they are definitely worth the price.</p>
+          <h3>For a lack of better words, Mechanical Keyboards are just absolutely more satisfying to use. But thats not all.</h3>
+          <p>There are a ton of other cool features included in mechanical keyboards, so we'll go more in-depth and talk about each one. We listed out 6 reasons why we think mechanical keyboards are better for gaming.</p>
+        </div>
+          <img class="feature-img" src="images/featureImage-Keeb.jpg" alt="Keychron-K12-image-on-desk">
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html]: The `img` tag should be self-closing.

File: FE-101/fe101-article-page.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+            <p class="article-item-text">Make sure when purchasing a mechanical keyboard that they have programmable keys, because not all keyboards have this feature. It usually requires the keyboard to use special software and certain companies have better software than others. Before purchasing a keyboard with programmable keys, do some research to make sure the software is reliable and easy to use or else it can be a headache to set up.</p>
+          </div>
+          <div class="article-item">
+            <h2 class="article-item-header">2. Extremely Durable</h2>
+            <p class="article-item-text">Mechanical keyboards are known for their incredible durability and superior build quality when compared to the traditional rubber dome/membrane style keyboards. For mechanical keyboards each key gets its own independent switch, so for each keystroke there is a dedicated switch to register the movement. The way these keyboards are built allows for a very consistent and responsive gaming experience.</p>
+            <img class="article-item-img" src="images/durability-test.jpg" alt="Durability test for mechanical keyboard">
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html]: The `img` tag should be self-closing.

File: FE-101/fe101-article-page.html [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+            <p class="article-item-text">N-key rollover is incredibly useful for gaming. In the middle of a game, you want to make sure all of your keystrokes register so you're able to operate the controls in the most accurate way possible. If only half your keystrokes are actually taken by the system, your performance will undoubtedly suffer. N-key rollover is also very useful for people who are fast at typing and get up to a speed where they need to press two keys at virtually the same time. N-key rollover makes typing more enjoyable experience when you know the keys will always register.</p>
+            <img class="article-item-img" src="images/NkeyRollover.jpg" alt="Pressing Multiple keys">
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html]: The `img` tag should be self-closing.

File: FE-101/fe101-article-page.html [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+          </div>
+          <div class="article-item">
+            <h2 class="article-item-header">5. Customize Switches for Competitive Advantage</h2>
+            <p class="article-item-text">As we briefly mentioned before, there are a ton of different mechanical keyboard switches to choose from. You are able to choose between tactile, linear, clicky, heavy, and light switches.</p>
+            <img class="article-item-img" src="images/customizable3.jpg" alt="Pressing Multiple keys">
+            <p class="article-item-text">If you enjoy getting feedback from your keyboard you can use a tactile switch. If you like a smooth keypress, you can use linear switches. If you tend to press down hard on your keys, you can use a heavy switch to help protect your fingers from impact. There are a ton of different ways to customize your keyboard with different switch types.</p>
+            <img class="article-item-img" src="images/customizable2.jpg" alt="Pressing Multiple keys">
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html]: The `img` tag should be self-closing.

File: FE-101/fe101-article-page.html [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+                  <td><a href="https://thekey.company/products/c3-tangerine-switches-r2" target="_blank">Tangerine V2</a></td>
+                  <td>Linear</td>
+                  <td>62 cN</td>
+                </tr>
+              </tfoot>
+            </table>
+          </div>
+          <div class="article-item">
+            <h2 class="article-item-header">6. As loud as you (don't) want</h2>
+            <img class="article-item-img" src="images/customizable4.jpg" alt="Pressing Multiple keys">
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html]: The `img` tag should be self-closing.

File: FE-101/fe101-article-page.html [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+            <img class="article-item-img" src="images/customizable1.jpg" alt="Pressing Multiple keys">
```
Comment (lycoangelo) [inline FE-101/fe101-article-page.html]: The `img` tag should be self-closing.

**Review by lycoangelo — CHANGES_REQUESTED** [review]
Hi @nicoRDL, I wrote a couple of minor change requests:
- Please update all `img` tags to self-closing.
- Some indentations are off.
- Also, the long line of text can be formatted into multiple lines and will still render the same in the front end. I suggest using a prettier or any kind of formatting tool to make it more readable.

File: FE-101/main.css [hunk 8]
```diff
@@ -0,0 +1,335 @@
...
+}
+
+.footer {
+    color:#808080;
+    text-align: center;
+    font-weight: 700;
+    font-size: .875rem;
+    text-transform: uppercase;
+    margin: 1rem auto;
+}
```
Comment (lycoangelo) [inline FE-101/main.css:346]: Please change indentation to spaces only.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

**Comment by nicoRDL** [comment]
@lycoangelo noted on all code updates and fix suggestions. Applied all updates 84debb3d57b3c66e77da6d1d2b5d02e3066873f6. Requesting for re-review.

84debb3d57b3c66e77da6d1d2b5d02e3066873f6

**Comment by lycoangelo** [comment]
CLGTM! 👍🏻

**Review by lycoangelo — APPROVED** [review]

**Comment by lycoangelo** [comment]
@nicoRDL, I just realized that I don't have merge access on the repo.

**Comment by nicoRDL** [comment]
@lycoangelo I'll rectify merge access ASAP. Will ping you immediately once done ✌️

**Pull request merged**